### ISSUE TITLE
Make tab ordering in dashboard structure explicit.

### ISSUE THREAD

**Issue by dennisoelkers** [Graylog2/graylog2-server#12834]
Currently, the tab ordering of a dashboard is determined by the order of the `queries` property of the underlying search. This is error-prone and should be replaced with an explicit ordering in the underlying `view` instead.
